tests/misc_env/install_prereq.py

```diff
@@ -231,7 +231,7 @@ def setup_subscription_manager(
             # you can push content to the staging CDN through the Errata Tool,
             # and then test it with --baseurl=cdn.stage.redhat.com.
             config_ = get_cephci_config()
-            command = "sudo subscription-manager register --force "
+            command = "sudo subscription-manager clean; sudo subscription-manager register --force "
```

Comment:
I believe we are already doing `subscription-manager clean` as part of inventory... There is a planned activity to improve the reliability of this script.